src/plugins/intel_cpu/src/utils/ordered_map.hpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2018-2022 Intel Corporation
```

Comment:
As we discussed offline: we either should get rid of it or clearly state why standard containers cannot be used.

Reply:
Ordered map class removed.